tests/JsonLd/ContextBuilderTest.php

```diff
@@ -67,6 +67,27 @@ public function testResourceContext()
         $this->assertEquals($expected, $contextBuilder->getResourceContext($this->entityClass));
     }
 
+    public function testIriOnlyResourceContext()
+    {
+        $this->resourceMetadataFactoryProphecy->create($this->entityClass)->willReturn(new ResourceMetadata('DummyEntity', null, null, null, null, ['iri_only' => true]));
+        $this->propertyNameCollectionFactoryProphecy->create($this->entityClass)->willReturn(new PropertyNameCollection(['dummyPropertyA']));
+        $this->propertyMetadataFactoryProphecy->create($this->entityClass, 'dummyPropertyA')->willReturn(new PropertyMetadata(new Type(Type::BUILTIN_TYPE_STRING), 'Dummy property A', true, true, true, true, false, false));
+
+        $contextBuilder = new ContextBuilder($this->resourceNameCollectionFactoryProphecy->reveal(), $this->resourceMetadataFactoryProphecy->reveal(), $this->propertyNameCollectionFactoryProphecy->reveal(), $this->propertyMetadataFactoryProphecy->reveal(), $this->urlGeneratorProphecy->reveal());
+
+        $expected = [
+            '@vocab' => '#',
+            'hydra' => 'http://www.w3.org/ns/hydra/core#',
+            'hydra:member' => [
+                '@type' => '@id'
+            ],
+        ];
+
+        dump($contextBuilder->getResourceContext($this->entityClass));
```

Comment:
remove this

Reply:
already did in https://github.com/api-platform/core/pull/3454/commits/280ae045dc424b81750bb5e0aab346f807fb253b#diff-acf61da8ac68c0eec26247b10f067e27